src/platform/3ds/gui-font.c

```diff
@@ -3,35 +3,42 @@
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 #include "util/gui/font.h"
 #include "util/gui/font-metrics.h"
 #include "util/png-io.h"
 #include "util/vfs.h"
 #include "font.h"
-
-#include <sf2d.h>
+#include "ctr-gpu.h"
 
 #define CELL_HEIGHT 16
 #define CELL_WIDTH 16
 #define GLYPH_HEIGHT 12
 
 struct GUIFont {
-	sf2d_texture* tex;
+	struct ctrTexture texture;
 };
 
 struct GUIFont* GUIFontCreate(void) {
 	struct GUIFont* guiFont = malloc(sizeof(struct GUIFont));
-	if (!guiFont) {
-		return 0;
+	if (guiFont == NULL) {
+		return NULL;
```

Comment:
No real reason to change this; it's equivalent to what it was, and I don't like code churn without functional changes.